src/components/button.scss

```
@@ -1,5 +1,6 @@
 :root {
   --rm-mobile-min-height: 44px;
```

This number seems really high. I had to override this on LandOne because it made the whole concept of button--small redundant.

---

```suggestion
  --rm-mobile-tap-area: 44px;
```

On mobile buttons need to have at least a 44px by 44px size for adequate finger tap area. It could be a bit large for desktops, where something like `button--small` works with no problem. However `button-small` on mobile would have to be 44px also.

---

@jdmcleod This is the recommended min height for mobile devices. In the latest version of the buttons (that I just pushed) we define these as
```
--rm-button-height-small:  calc(6 * var(--rm-space-2x-small)); // ~ 24px
--rm-button-height-medium: calc(8 * var(--rm-space-2x-small)); // ~ 32px
--rm-button-height-large:  calc(11 * var(--rm-space-2x-small)); // ~ 44px
```

Which also solves @zoopmaster comment. and makes the defaults smaller

On Mobile, buttons will resize to `--rm-button-height-large` to ensure the tap area is large enough.